source/layouts/partials/_header.erb

```diff
@@ -80,10 +80,6 @@
                     </a>
                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                       <a class="dropdown-header">Exchanges</a>
-                        <!--
-                          <a class="dropdown-item" href="https://bitshares.openledger.info/#/market/OPEN.GRC_OPEN.BTC"><i class="fa fa-exchange"></i> OpenLedger</a>

```

Comment:
Can we keep this part commented out?  It is feasible that OpenLedger will be usable again if they ever fix their gateway.